cmd/auth.go

```diff
@@ -34,18 +34,24 @@ var authCmd = &cobra.Command{
 	Long: fmt.Sprintf(`Command maps Snyk Token with UUID and Outputs 'crda_key' for further Authentication.
 	
 	To get "Snyk Token" Please click here: %s`, snykURL),
-	Run: main,
+	RunE: runAuth,
 }
 
 func init() {
 	rootCmd.AddCommand(authCmd)
-	authCmd.Flags().StringVarP(&snykToken, "snyk-token", "t", "", "Authenticate with Snyk Token to unlock Verbose stack anaylses.")
+	authCmd.Flags().StringVarP(&snykToken, "snyk-token", "t", "", "Authenticate with Snyk Token to unlock Verbose stack analyses.")
 }
 
-func main(cmd *cobra.Command, args []string) {
+// runAuth is controller for Auth command.
+func runAuth(cmd *cobra.Command, _ []string) error {
 	log.Debug().Msgf("Executing Auth command.")
+	var err error
 	if snykToken == "" {
-		snykToken = promptForToken()
+		snykToken, err = promptForToken()
+		if err != nil {
```

Comment:
Where do we send snyk token association telemetry event?
i.e. we should have events to indicate user added snyk account or not.

Reply:
There is no such exclusive event for Snyk token association. Do you want me to a Property to `crda auth` event itself...?

Reply:
Basically successfully validated snyk token against our backend

Reply:
Ack.